tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -107,22 +107,28 @@ public:
       static constexpr std::size_t kOCNameMaxLength = 64;
    };
 
+   struct RDaosRequest {
```

Comment:
Maybe rename to `RAkeyRequest` (?) - to avoid confusion with the (growing) number of other classes, i.e. `FetchUpdateArgs`, `RWRequest`, ...

Also note that there is no `Daos` prefix, as this class' qualified name is already `RDaosObject::RAkeyRequest`.